adoc_event_tables/cache_spec.adoc

```diff
@@ -36,8 +36,10 @@
 |CACHE.L2.PREF.MERGE |L2 cache hits caused by prefetcher where data is not yet in cache but was already requested by preceding miss.
 |CACHE.L2.PREF.UNUSED |Number of cachelines brought into L2 by prefetcher and evicted without being accessed even once.
 |CACHE.L2.WB |Writebacks to next level cache or memory.
-|CACHE.SNOOP.LOCAL_REQ_REMOTE_HITM.SPEC |Private cache misses where data was found in another core cache in modified state. This event can be used to accout for contested accesses cases where several cores read/write the same cachelines.
-|CACHE.SNOOP.REMOTE_REQ_LOCAL_HITM |Snoop requests which found cacheline in the core cache in modified state. This event can be used to accout for contested accesses cases where several cores read/write the same cachelines.
+|CACHE.SNOOP.LOCAL_REQ_REMOTE_HITM.SPEC |Private cache misses where data was found in another core cache in modified state. This event can be used to account for contested accesses cases where several cores read/write the same cachelines.
```

Comment:
One thing to keep in mind with this event is that it may only reflect HITMs in a subset of caches.  E.g., if the L3/LLC is external, this may only count HITMs in the L1D and L2.  I guess the implication is that LOCAL is only guaranteed to cover caches local to (within) the hart.  I think that's okay, the kind of sharing cases we're looking for are likely to be getting HITMs in the L1D's of the conflicting harts.